src/ReactNativeSVG.web.ts

```diff
@@ -247,7 +251,40 @@ export class WebShape<
     super(props, context);
     SvgTouchableMixin(this);
     this._remeasureMetricsOnActivation = remeasure.bind(this);
+    const componentDidMount = this.componentDidUpdate as () => void;
+    this.componentDidMount = () => {
+      componentDidMount();
```

Comment:
throws because you are not passing props.

Also: Why rename componentDidUpdate as componentDidMount?